### DISCUSSION TITLE
9.14.1: Constant net::ERR_CONNECTION_RESET 200 (OK) Errors from the SDK

### DISCUSSION THREAD

**Discussion by faulander** [directus/directus#14368] [Archive]
### Describe the Bug

![image](https://user-images.githubusercontent.com/38588197/178291578-65c8b3f0-6298-4e83-9e46-432f3a66ecfb.png)

Directus and the SDK consuming program are running on the same DOCKER instance, sharing the same network interface.
I restarted the containers,
i restarted docker,
i restarted the host.

I also changed the "SERVER_KEEP_ALIVE_TIMEOUT" to 30000 but it didn't help.

EDIT: This only happens since upgrading to 9.14.1, i didn't have that behaviour before.

### To Reproduce

i cannot reproduce the connection issue, how should i do that?

### What version of Directus are you using?

9.14.1

### What version of Node.js are you using?

16.13.1

### What database are you using?

PG 13.3-1.pgdg100+1

### What browser are you using?

doesn't matter

### How are you deploying Directus?

Docker

**Comment by rijkvanzanten** [comment]
I'm pretty sure this is a configuration problem in how the Docker containers are interacting. Neither the API nor the SDK will throw that error, as it's a generic browser error 🤔

**Comment by faulander** [comment]
the system was running for months before upgrading to 9.14.1
I wonder what could cause this?

**Comment by azrikahar** [comment]
> This only happens since upgrading to 9.14.1, i didn't have that behaviour before.

What version did you upgrade from? It does feel like an external issue (browser, browser extensions, proxy), but perhaps you can try just using the REST API (not the SDK) and see will you face the same issue to at least rule things out.
